src/components/Menus/Sidebar/Sidebar.test.js

```diff
@@ -43,19 +43,11 @@ describe("When project has images", () => {
     );
   });
 
-  test("File pane closed by default", () => {
-    expect(screen.queryByTitle("sidebar.expand")).toBeInTheDocument();
-  });
-
-  test("Clicking expand opens the file pane", () => {
-    const expandButton = screen.getByTitle("sidebar.expand");
-    fireEvent.click(expandButton);
-    expect(screen.queryByText("filePanel.files")).toBeInTheDocument();
```

Comment:
Could we still have this test, but open the sidebar first, then close it, like the reverse of what we did before with the collapse button?

Reply:
I thought the same then realised that's already tested below

Reply:
Which line is this on? I can't see it 😅

Reply:
checking the menu closes is line 50 which I think is the same as the test you're suggesting

Reply:
Ah no sorry, I was suggesting a test that the sidebar opens when you click the expand button (but if we're defaulting the sidebar to open then we'll have to close it first) 😅